examples/securenn/network_c.py

```diff
@@ -116,8 +113,7 @@ def loop_body(i: tf.Tensor, max_iter: tf.Tensor, nb_epochs: tf.Tensor, avg_loss:
             layer2 = pooling(tf.nn.relu(conv2d(layer1, Wconv2, self.STRIDE) + bconv2))
             layer2 = tf.reshape(layer2, [-1, self.HIDDEN_FC1])
             layer3 = tf.nn.relu(tf.matmul(layer2, Wfc1) + bfc1)
-            layer4 = tf.nn.relu(tf.matmul(layer3, Wfc2) + bfc2)
-            logits = tf.matmul(layer4, Wfc3) + bfc3
+            logits = tf.matmul(layer3, Wfc2) + bfc2
```

Comment:
any reason we've removed the penultimate layer here?  [this diagram](https://www.google.com/url?sa=i&source=images&cd=&cad=rja&uact=8&ved=2ahUKEwieqbngr97fAhXCTd8KHYkpBX4QjRx6BAgBEAU&url=https%3A%2F%2Fwww.researchgate.net%2Ffigure%2FThe-LeNet-5-Architecture-a-convolutional-neural-network_fig4_321586653&psig=AOvVaw0NjyTeTVArGQsqyznFd80v&ust=1547043357750404) suggests the network should have 2 linear layers before the output layer

Reply:
ah, just checked the SecureNN paper -- looks like they removed the penultimate layer from LeNet, nvm :)